ui/settings.ts

```diff
@@ -36,6 +37,7 @@ export class Settings {
         // rant start: Why is typescript so stupid as to not see that in the
         // function this.reset() the following 2 attributes are set?
         this.include_crypto2crypto = false;
+        this.include_gas_costs = true;
```

Comment:
And then move this inside the `reset()` function below so that it gets reset when a user logs out.

Reply:
Now that I think of it, you can do the same with `include_crypto2crypto`. This way we avoid the double instantiation (both in ctor and in the `reset()` function).

Reply:
Good catch.  For the `crypto2crypto` attribute, it looks like the constructor is setting it to `false` but the reset is setting to `true`.  Are you thinking remove it from the constructor and have it reset to `true`?

Reply:
Default should be true, so yes it should be reset to true.

Reply:
Just pushed some fixes.  I also moved `taxfree_after_period` to work the same way since it was the only other variable getting set in the constructor and then immediately reset.  Let me know if that was what you were thinking.